packages/shared/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@redhat-cloud-services/javascript-clients-shared",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./index.js",
+  "module": "./esm/index.js",
+  "typings": "./index.d.ts",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/RedHatInsights/javascript-clients.git"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/RedHatInsights/javascript-clients/issues"
+  },
+  "homepage": "https://github.com/RedHatInsights/javascript-clients#readme",
+  "scripts": {
+    "doc": "typedoc --theme markdown --out doc --gitRevision master index.ts"
+  },
+  "dependencies": {
+    "axios": "^0.27.2",
```

Comment:
We want to use axios `1.x` from now on. The old versions have many security vulnerabilities.

Reply:
I'll make a story to update all client's `axios` version since all of them are out of date